appengine/flexible/tasks/appengine_queues/app_engine_queue_snippets.py

```diff
@@ -0,0 +1,144 @@
+# Copyright 2016 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import argparse
+import base64
+import datetime
+
+from googleapiclient import discovery
+
+
+def format_rfc3339(datetime_instance):
+    """Format a datetime per RFC 3339."""
+    return datetime_instance.isoformat("T") + "Z"
+
+
+def get_seconds_from_now_rfc3339(seconds):
+    """Return seconds from the current time as a RFC 3339 string."""
+    d = datetime.datetime.utcnow() + datetime.timedelta(seconds=seconds)
+    return format_rfc3339(d)
+
+
+def list_queues(api_key, project_id, location_id):
+    """List the queues in the location."""
+    client = get_client(api_key)
+    parent = 'projects/{}/locations/{}'.format(project_id, location_id)
+    queues = []
+    next_page_token = None
+
+    while True:
+        response = client.projects().locations().queues().list(
+            parent=parent, pageToken=next_page_token).execute()
+        queues += response['queues']
+        if next_page_token is None:
+            break
+
+    print('Listing queues for location {}'.format(location_id))
+
+    for queue in response['queues']:
+        print queue['name']
```

Comment:
print is a function.

Reply:
Done (I also added from __future__ import print_function as I prefer to include that for reasons I've mentioned previously)